Python/Python.sublime-syntax

```diff
@@ -700,18 +729,34 @@ contexts:
 
   function-call-wrapper:
     - meta_scope: meta.function-call.python
-    - match: '{{identifier}}'
-      scope: meta.qualified-name.python variable.function.python
     - match: \)
       scope: punctuation.section.arguments.end.python
       set: after-expression
     - match: \(
-      scope: punctuation.section.arguments.begin.python
+      scope: meta.function-call.python punctuation.section.arguments.begin.python
       push:
+        - clear_scopes: 1
         - meta_content_scope: meta.function-call.arguments.python
```

Comment:
Most `meta.`s like `meta.group`, `meta.brackets`, ... or even `string.quoted` scopes include the opening and closing punctuation. The C++ syntax as an example uses `meta.group` to scope the function-call arguments. 

I am just wondering why `meta.function-call.arguments` is handled differently (not including the parens)?

Reply:
Any thoughts on including the parens in the `.arguments` specialization?

Reply:
I probably did it because of `set` semantics and that requiring me to push into a temporary context to get rid of the previous `meta_scope`. See 5be7b83dc88c0dc0db1fe0292e1756616a780938 for details.